modules/ROOT/pages/sync/adding-sync-to-your-app.adoc

```diff
@@ -9,6 +9,94 @@ This section will help you to set up the {sync-service} service SDK in your App.
 [role="primary"]
 .Android
 ****
+```
+implementation "org.aerogear:aerogear-sync:1.1.0-SNAPSHOT"
+```
+****
+
+[role="secondary"]
+.iOS
+****
+****
+
+[role="secondary"]
+.Cordova
+****
+****
+
+[role="secondary"]
+.Xamarin
+****
+****
+
+== Setting Up the Sync Mobile Service
+
+The Sync SDK is using the link:https://github.com/apollographql/[Apollo GraphQL Client] behind of the scenes to handle link:http://graphql.github.io/[GraphQL]. 
+
+link:https://github.com/apollographql[Apollo client] requires some configuration to work.
+
+[role="primary"]
+.Android
+****
+1. Create a special folder `graphql` inside of `src/main`
+2. Create a file `schema.json` with your GraphQL schema
+3. Create a file with name that corresponds objects you want to create and the `.graphql` extension
+4. Add all your queries/mutations/subscriptions to the `.graphql` file. Apollo will generate the appropriate Java classes with nested classes for reading from the network response. 
+
+For example if you create a file `graphql/org/aerogear/android/app/memeolist/graphql/myProject.graphql` with the content bellow:
+
+```
+query myNotes {
+  allNotes() {
+    id
+  }
+}
+```
+
+Apollo will create a file `MyNotesQuery` to you fire your query.
+****
+
+[role="secondary"]
+.iOS
+****
+****
+
+[role="secondary"]
+.Cordova
+****
+****
+
+[role="secondary"]
+.Xamarin
+****
+****
+
+## Using the Sync Mobile Service
+
+Client SDK provides a simple way to get the Apollo client configured to use.
+
+[role="primary"]
+.Android
+****
+```
+SyncService
+        .getInstance()
+        .getApolloClient()
+        .query(MyNotesQuery.builder().build())
+        .enqueue(new ApolloCall.Callback<MyNotesQuery.Data>() {
```

Comment:
People will get error on MyNotesQuery object if they not follow schema above. Best to mention that `MyNotes` is generated from example schema above (in the note)